src/Microsoft.AspNetCore.Blazor/Components/Parameter.cs

```diff
@@ -11,31 +9,20 @@ namespace Microsoft.AspNetCore.Blazor.Components
     /// </summary>
     public readonly struct Parameter
     {
-        private readonly RenderTreeFrame[] _frames;
-        private readonly int _frameIndex;
-
-        internal Parameter(RenderTreeFrame[] frames, int currentIndex)
-        {
-            _frames = frames;
-            _frameIndex = currentIndex;
-        }
-
         /// <summary>
         /// Gets the name of the parameter.
         /// </summary>
-        public string Name
-            => _frames[_frameIndex].AttributeName;
+        public string Name { get; }
 
         /// <summary>
         /// Gets the value of the parameter.
         /// </summary>
-        public object Value
-            => _frames[_frameIndex].AttributeValue;
+        public object Value { get; }
```

Comment:
Now they are arguments tho

Reply:
Fair point. Rephrased in such a way as to still avoid using the word "argument" 😼

Reply:
Good. I'd like to avoid an argument about arguments.